src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -73,14 +73,14 @@ $_stacked-reduce-width: 20px;
   position: relative;
   width: 100%;
   aspect-ratio: 4 / 3;
-}
 
-.event-list-item__image {
-  width: 100%;
-  height: 100%;
-  object-fit: cover;
-  position: relative;
-  z-index: $z-10;
+  * {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    position: relative;
+    z-index: $z-10;
+  }
```

Comment:
Please consider if this change is needed.

I do not understand the need for this change. It does not seem to be related to the change at hand, it is not in a separate commit and it is not mentioned in the commit comment or in the pull request.

In general I would also think that a wildcard child selector would be something that we would want to avoid.

Reply:
I get the confusion - I actually think this was meant to be in another PR

Either way - I've pulled it out into its own commit now, along with a code comment on why * is needed.